### ISSUE TITLE
Sync Imap account

### ISSUE THREAD

**Issue by GP-S** [twentyhq/twenty#917]
Good feature available in hubspot for instance is the ability to synchronize your email account and attach mails or conversation from a customer to an opportunity/company account. 

It can be down by adding a kind of inbox for all messages and some auto filters on the sender and recipient field of the mail.

Main idea is to store discussion/attached file directly in the crm without need to copy paste your mail conversation. 

P.S: Of course it can be extended to linkedin, whatsapp ...

**Comment by FelixMalfait** [comment]
Yes we should definitely build that, I agree. I had logged in a similar idea in the discussion section: https://github.com/twentyhq/twenty/discussions/211

First step would be to implement a new type of activity (currently we only have notes, then tasks will come this week, a we probably need a third category that describes a past activity I want to log). Then leverage the google sign-in to import email through this as a first step. Then offering a more generic imap solution would be great.

I would say this is on the medium-term roadmap so something that is very likely to be fully implemented before end of year, but not likely to be implemented in the next 5 weeks.
